server_2x/views/layout.ejs

```diff
@@ -59,12 +59,12 @@
             </li>
             <li class="nav-item">
                 <a title="Davids super power tool" class="nav-link text-light text-center btn-sm d-none"
-                   id="menuListView" href="/mon/printerMap"><i class="fas fa-list"></i> Map</a>
+                   id="menuListView" href="/mon/map"><i class="fas fa-list"></i> Map</a>
```

Comment:
These need to be added to the changelog please

Reply:
Also this one should not have reached canary yet... stupid me and good catch!

Reply:
Reverted in #705